test/unit/org/apache/cassandra/index/sasi/SASIIndexTest.java

```diff
@@ -199,7 +200,7 @@ public void testSASIComponentsAddedToSnapshot() throws Throwable
 
         try
         {
-            store.snapshot(snapshotName);
+            SnapshotManager.instance.takeSnapshot(snapshotName, store.getKeyspaceTableName());
```

Comment:
not that it matters much, but the equivalent of this is actually:
```suggestion
            SnapshotManager.instance.takeSnapshot(snapshotName, store.getKeyspaceTableName(), store.getTableName());
```

and in fact you clear the snapshot after the test for the specific table

Reply:
not applicable